src/Common/src/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -236,6 +236,9 @@ public bool IsMarshallingRequired()
             return true;
         }
 
+        /// <summary>
+        /// Given a NativeType enum returns a corresponding Type that should be passed to the Native side
```

Comment:
You are right. This is a bug. It should return char* here for LPWStr. I have pushed a fix for it.